README.md

```diff
@@ -2,7 +2,7 @@ Unison File Synchronizer
 ========================
 
 [![Build Status](https://travis-ci.org/bcpierce00/unison.svg?branch=master)](https://travis-ci.org/bcpierce00/unison)
-![Main workflow](https://github.com/bcpierce00/unison/workflows/Main%20workflow/badge.svg)
+[![Main workflow](https://github.com/bcpierce00/unison/workflows/Main%20workflow/badge.svg)](https://github.com/bcpierce00/unison/actions?query=workflow%3A+Main+Workflow")
```

Comment:
Why is there a trailing "?  It seems spurious, even though I don't know the URL format, and if I remove it, the link still works.

Other than that looks good to me.

Reply:
I''m not sure which ? you refer to. The intended target is a filtered list.

Reply:
I think we are having character set issues.    In the commit you want to merge, the URL for the workflow ends with the string Workflow followed by double-quote character.  There is no opening quote and I don't know why this one is there.

Reply:
updated. thx for correcting